### PR TITLE
Add script to create an example Cloud Monitoring dashboard and README

### LINKED ISSUE #42: Add script to create example dashboard in Cloud Monitoring
In order for developers to confirm easily whether the example is working as expected, add a script to create a Cloud Monitoring dashboard.

### PR DESCRIPTION
**Pull request by ymotongpoo**
* Fixes #42
* Add README for monitoring example (#23)

### CODE REVIEW

File: example/metric/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+$ go build -o metrics
+$ ./metrics
+2020/06/11 21:11:15 Most recent data: counter 110, observer 13.45
+2020/06/11 21:11:15 Most recent data: counter 160, observer 16.02
+2020/06/11 21:11:15 Most recent data: counter 134, observer 14.33
+2020/06/11 21:11:15 Most recent data: counter 125, observer 15.12
+...
+```
+
+In order to have enough amount of metrics to create charts in the dashboard for Counter and Observer values in the example, keep the application running at least for 5 min before start creating the dashboard.
```
Comment (nilebox) [inline example/metric/README.md:27]: nit: "running for 5 minutes before creating a dashboard" looks easier to read.

File: example/metric/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+```
+$ ./create_dashboard.sh
+```
+
+This script creates a dashboard titled "OpenTelemetry exporter example/metric".
+
+You'll find the line charts like below once you create the dashboard.
+
+<img width="1200" alt="2 charts in dashboard" src="images/charts.png?raw=true"/>
```
Comment (nilebox) [inline example/metric/README.md:47]: This should be wrapped into a code block (```)

Comment (ymotongpoo) [reply example/metric/README.md:47]: Should we? `` tag in markdown shows images without any issues.
c.f.: https://github.com/GoogleCloudPlatform/opentelemetry-operations-js/blob/master/samples/trace/README.md

**Comment by ymotongpoo** [comment]
Thank you @nilebox and @james-bebbington for the review. Especially the wording in English documents is one of my challenges, so the review helps me a lot!

**Pull request merged**